diagnostic.md

```diff
@@ -14,6 +17,15 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
+$( "li" ).each(function( index ) {
+  console.log( index + ": " + $( this ).text() );
+});
+
+$( "li" ).each(function( index ) {
+  console.log( index + ": " + $( this ).text() );
+});
+
```

Comment:
These two answers are the same and the call to `.each()` is unnecessary.  From `<ol>`, two possible ways to select `<li>` elements are `$('ol > li');` and `$('ol').children();`.  Some other possible ways are `$('li');`, `$('body').find('li');`, and `$('body').children('ol').children('li');`.